README.md

```diff
@@ -128,6 +129,14 @@ We also recommend these lists.
 * [WebService::DigitalOcean](https://metacpan.org/pod/WebService::DigitalOcean) - Access the DigitalOcean RESTful API (v2)
 * [WebService::Dropbox](https://metacpan.org/pod/WebService::Dropbox) - Interface to Dropbox API
 
+## Commercial WebServices
+
+* [Net::Xero](https://metacpan.org/pod/Net::Xero) - Interface to Xero accounting
+* [WebService::Spotify](https://metacpan.org/pod/WebService::Spotify) - A simple interface to the Spotify Web API
+* [WebService::Xero](https://metacpan.org/pod/WebService::Xero) - Access Xero Accounting Package Public and Private Application API
+* [WWW::Shopify](https://metacpan.org/pod/WWW::Shopify) - object representing acess to a particular Shopify store
+* [WWW::Spotify](https://metacpan.org/pod/WWW::Spotify) - Spotify Web API Wrapper
```

Comment:
Perhaps it makes confusing if it suggests multiple Spotify's clients. So I suppose that we should show only one client.
How do you think? There is some difference between them?

Reply:
I couldnt say. They both seem current.

Reply:
Okay, I see.

Reply:
Okay, I see.